pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```

Comment:
You'll either have to enable the upstream hook, or disable testing the kube rbac kinds. Or disable the rbac authorizer, maybe. Are there other options?